gui/src/components/OnboardingWizard/OnboardingWizard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, Button, Stack, Typography } from "@mui/material";
+import { invoke } from "@tauri-apps/api/tauri";
+import { useState } from "react";
+
+import { useKeyPress } from "../../hooks";
+import { useWizardForm } from "../../store/wizard";
+import { Logo } from "../Logo";
+import { OnboardingCarousel } from "./OnboardingCarousel";
+import { steps } from "./Steps";
+
+interface Props {
+  closeOnboarding: () => void;
+}
+
+export function OnboardingWizard({ closeOnboarding }: Props) {
```

Comment:
I noticed you added everything inside a folder `OnboardingWizard`, but I think it would be best to push everything to the `components` folder and use the prefix `Onboarding` to maintain consistency with the rest of the application.

Reply:
Actually I was in the process of breaking that consistency for all the Settings* stuff, because it was making my fuzzy finding a pain to use

Reply:
👍 it's fine by me, but in that case, be very careful where you place components, and maybe don't allow for another level of nesting. It's really easy for nesting to become a mess.